CODIGO/Declares.bas

```diff
@@ -1083,6 +1098,8 @@ Public Const DISTANCIA_ENVIO_DATOS As Byte = 3
 Public cooldown_ataque As New clsCooldown
 Public cooldown_hechizo As New clsCooldown
 
+Public LobbyList() As t_LobbyData
```

Comment:
instead of adding a global array we can just pass the parameter to the form using this